### PR TITLE
Set Tax included as default value for amounts in cart rule form.

### LINKED ISSUE #28165: Cart rules should have their amount set to `Tax included` by default
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

When creating a cart rule an selecting a reduction amount, this amount is by default `Tax excluded`. This is not logical. 
If a cart rule has a reduction amount, it should not have any tax. A reduction using a voucher is like money and money does not have taxes, only products sold.

Anyway, if a cart rule gives a reduction tax excluded, this means that it is increased by the amount of the tax. Thus the tax will be a surprise to the merchant. This is most of the time not the expected behavoir.

### Describe the solution you'd like

In the cart rules form, `Tax included` should be the default value.

### Alternatives you've considered

One must be very careful, and change `Tax excluded` into `Tax included` for every cart rule that gives a reduction amount.

### Additional context

_No response_

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

### PR DESCRIPTION
**Pull request by lmeyer1**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | A new cart rule should have its amounts specified as `Tax included` by default
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28165.
| Related PRs       | None
| How to test?      | Create a new cart rule and notice that the default values are `Tax included`, both on the `Conditions` and the `Actions` tab.
| Possible impacts? | None

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28166)

### CODE REVIEW

**Comment by hibatallahAouadni** [comment]
**Friendly reminder: _we need the PM approval on the linked issue!_**

**Comment by matthieu-rolland** [comment]
lgtm, let's wait for PM validation

**Comment by lmeyer1** [comment]
@matthieu-rolland The linked issue is approved by @camilleganzin, if you mean this by `PM validation`

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
@hibatallahAouadni it seems that @camilleganzin has approved.

**Review by AureRita — APPROVED** [review]
Hi @lmeyer1 

Thank you for your PR, I tested it and it works, as we can see here :

https://user-images.githubusercontent.com/100699445/163974258-782fc559-cc6a-478a-8592-a871867529f6.mp4

**Comment by Progi1984** [comment]
Thanks @lmeyer1 & @AureRita

**Pull request merged**
